bower.json

```diff
@@ -17,10 +14,9 @@
     "output",
     "test",
     "bower.json",
-    "gulpfile.js",
     "package.json"
   ],
   "dependencies": {
-    "purescript-prelude": "^0.1.0"
+    "purescript-prelude": "^1.0.0"
```

Comment:
I think we should bump to 1.0, but check each repo for outstanding PRs and issues which should be fixed as we go.